jekyll/_cci2/github-app-trigger-options.adoc

```diff
@@ -0,0 +1,83 @@
+
+Once your project is set up you can create additional pipelines and triggers (menu:Project settings[Triggers]). You can configure your triggers to initiate pipelines on a variety of events, including non-push events, for example, a pull request being opened, or refined push events, for example, only pushes to a default branch, or pushes to a tag.
+
+== Scope of push event support
+
+* Pipelines are not triggered on branch or tag deletion push events, even when you have the "All push" trigger option selected.
+* Pipelines are not triggered when a new branch is created if there are no new commits on the branch.
+
+== Supported trigger events
+
+GitHub App triggers allow you to define the work to do (pipeline) when an event takes place (trigger). GitHub App triggers are an orchestration tool available in CircleCI. You can access GitHub App triggers using the "Run on" menu when setting up a trigger. The following preset trigger options are available:
+
+* *All pushes*: Trigger your pipeline on all pushes to your repo. All pushes corresponds to the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads#push[GitHub App push event]. No pipelines are triggered on push events when a branch or tag are deleted. No pipeline is triggered when a branch is created.
+* *Tag pushes*: Trigger a pipeline on pushes to tags only.
+* *Pushes to default branch*: Trigger a pipeline on pushes to the default branch only.
+* *PR opened or pushed to, default branch pushes, tag pushes*: Trigger a pipeline on pushes to a branch for which there is an open Pull Request (PR), when a PR is opened, pushes to any tag, and pushes to the default branch.
+* *PR opened*: Trigger a pipeline when a PR is opened only. See the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=opened#pull_request[GitHub docs for the pull request opened event].
+* *PR merged*: Trigger a pipeline when a PR is merged only. See the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=closed#pull_request[GitHub docs for pull request closed].
+* *PR marked ready for review*: Trigger a pipeline when a PR is taken out of draft mode and marked ready for review. See the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=review_requested#pull_request[GitHub docs for pull request review requested].
+* *"run-ci" label added to PR*: Trigger a pipeline when a label "run-ci" is added to a PR. See the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=labeled#pull_request[GitHub docs for pull request labelled].
+
```

Comment:
Can we add a version of this (sorry if you've added elsewhere, havent' read everything yet)

> Different trigger rules cannot be combined in a single trigger. However, you can create multiple triggers for the same pipeline, with each trigger using a different trigger rule.

Reply:
Added as an FAQ at the end